src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -117,7 +117,7 @@ module sonic-port{
 					
 				leaf mtu {
 					type uint16 {
-						range 1..9216;
+						range 68..9216;
```

Comment:
Please update unit test to cover this change.

Reply:
I have updated unit test and comment. Please review. @ganglyu

Reply:
unit test is updated, but where is the comment?

Reply:
I added it at the beginning.
We use the iproute2 tool to set the kernel MTU in portmgrd. If MTU value is less than 68, iproute2 tool will report an error.